src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -143,6 +149,21 @@ public ApiMethodView getApiMethodView(Interface service, Method method) {
     return apiMethodTransformer.generateMethod(methodContext);
   }
 
+  public List<PathTemplateView> getPathTemplates(Interface service) {
+    SurfaceTransformerContext context = getSurfaceTransformerContextFromService(service);
```

Comment:
This is intended to be very temp. Right now, I'm planning  the following PRs for full MVVM:
1. gRPC stubs
2. api methods
3. gRPC doc message
4. enums
5. finish